packages/modules/web_themes/koala/source/src/stores/mqtt-store.ts

```diff
@@ -941,6 +954,46 @@ export const useMqttStore = defineStore('mqtt', () => {
     });
   };
 
+  /**
+   * Get boolean value for DC charging enabled / disabled
+   * @returns boolean
+   */
+  const DCChargingEnabled = computed(() => {
+    return (getValue.value('openWB/optional/dc_charging') as boolean) || 0;
+  });
+
+  /**
+   * Get or set the charge point connected vehicle instant charging DC power identified by the charge point id
+   * @param chargePointId charge point id
+   * @returns number
+   */
+  const chargePointConnectedVehicleInstantDCChargePower = (
+    chargePointId: number,
+  ) => {
+    return computed({
+      get() {
+        const DCCurrent =
+          chargePointConnectedVehicleChargeTemplate(chargePointId).value
+            ?.chargemode?.instant_charging?.dc_current;
+        if (DCCurrent !== undefined) {
+          return (DCCurrent * 3 * 230) / 1000;
+        } else {
+          return 0;
+        }
```

Comment:
Exemplarisch für weitere Methoden:
```suggestion
        const dcCurrent =
          chargePointConnectedVehicleChargeTemplate(chargePointId).value
            ?.chargemode?.instant_charging?.dc_current;
        if (dcCurrent !== undefined) {
          return (dcCurrent * 3 * 230) / 1000;
        } else {
          return 0;
        }
```

Reply:
korrigiert

Reply:
korrigiert